### PR TITLE
Adress HTML violations

### PR DESCRIPTION
**Pull request by RoiEXLab**
Follow-up of https://github.com/NodeBB/nodebb-theme-persona/issues/346:
This adresses a couple of html violations of this theme

### CODE REVIEW

**Comment by RoiEXLab** [comment]
@julianlam I'm not quite sure what to do on the following lines:
[Userpicture](https://github.com/NodeBB/nodebb-theme-persona/blob/master/templates/partials/menu.tpl#L66-L67)
[Brand Logo](https://github.com/RoiEXLab/nodebb-theme-persona/blob/patch-1/templates/partials/menu.tpl#L9-L11)

In my opinion, those tags should be enclosed inside an if-endif-block, But I'm not sure if this would break things... i believe those "style" attributes are there on purpouse. Those tags are still causing bad html

**Comment by julianlam** [comment]
For user picture, look up another use case for the user picture. You are correct in that it should be wrapped in an `<!-- IF -->`.

[See here](https://github.com/NodeBB/nodebb-theme-persona/blob/master/templates/partials/account/header.tpl#L5-L9).

What issues are present in the brand logo?

**Comment by julianlam** [comment]
Oh... yes, should be safe to wrap the brand logo lines in a conditional as well, if `brand:logo` is not set.

**Comment by RoiEXLab** [comment]
@julianlam done!

**Comment by RoiEXLab** [comment]
Should I revert?

**Comment by pichalite** [comment]
Yes

**Comment by RoiEXLab** [comment]
Reverted change...

**Comment by RoiEXLab** [comment]
OK, this will be a task for another PR...

**Comment by RoiEXLab** [comment]
@pichalite This PR is still open, no changes were made to any of the logic.
As I see this, this PR can be merged without any problems.

File: templates/header.tpl [hunk 1]
```diff
@@ -39,7 +39,7 @@
 
 <body class="{bodyClass} skin-{config.bootswatchSkin}">
 	<nav id="menu" class="hidden">
-		<section class="menu-profile">
```
Comment (julianlam) [inline templates/header.tpl]: Out of curiosity, what the problem with `section` tag here?

File: templates/partials/categories/item.tpl [hunk 2]
```diff
@@ -9,9 +9,9 @@
 		<h2 class="title">
 			<!-- IMPORT partials/categories/link.tpl --><br/>
 			<!-- IF ../descriptionParsed -->
-			<div class="description">
+			<span class="description">
 			{../descriptionParsed}
-			</div>
+			</span>
```
Comment (julianlam) [inline templates/partials/categories/item.tpl]: Needs to be made block-level, since `span` is inline...

File: templates/header.tpl (hunk 1 above)
Comment (RoiEXLab) [reply templates/header.tpl]: A section element needs a h1-h6 tag as child:
https://www.w3.org/TR/html5/sections.html#the-section-element

Comment (julianlam) [reply templates/header.tpl]: Understood, so semantically it does not belong, okay :+1:

**Comment by RoiEXLab** [comment]
@julianlam Updated, span is inlined

File: templates/partials/categories/item.tpl [hunk 3]
```diff
@@ -9,9 +9,7 @@
 		<h2 class="title">
 			<!-- IMPORT partials/categories/link.tpl --><br/>
 			<!-- IF ../descriptionParsed -->
-			<span class="description">
-			{../descriptionParsed}
-			</span>
+			<span class="description">{../descriptionParsed}</span>
```
Comment (julianlam) [inline templates/partials/categories/item.tpl]: Well, not exactly... it was fine with the line breaks, but what I mean is a `div` is a block element (multiple divs stack on top of one another) while a span is inline (multiple spans are next to each other), so changing a div to a span isn't enough, you need to also apply the bootstrap class to make it a block element.

Comment (RoiEXLab) [reply templates/partials/categories/item.tpl]: @julianlam Sorry, my bad, was updated too, looks exactly the same now

**Comment by RoiEXLab** [comment]
@julianlam Bumping...

**Comment by julianlam** [comment]
Thanks, v1.6.0 is landing tomorrow so we'll re-visit afterwards.

**Comment by RoiEXLab** [comment]
@julianlam Bumping again, now that 1.6.0 is released, and 1.6.1 is in development

**Comment by RoiEXLab** [comment]
@julianlam Bumping again, now that 1.6.0 is released, and 1.6.1 is in development

**Comment by julianlam** [comment]
Thanks :smile:

**Comment by RoiEXLab** [comment]
No Problem!

**Pull request merged**